### ISSUE TITLE
Optimizer/Training Strategy

### ISSUE THREAD

**Issue by ZhengGang85129** [ZhengGang85129/DashCam#39]
## Fine-tune the model with multiple stages.
   1. Freeze all the parameters in r3d_18 layers except for Classifiers ( final FC layers). Choose a proper scheduler and optimizer (AdamW with weight decay or RAdam) with lr = 1e-4. Train the model for 10 epochs.
   2. Choose an appropriate checkpoint as the starting point(best ckpt for example). Unfreeze the parameters in deeper layer (ex: layer 4). Choose a proper scheduler and optimizer (AdamW with weight decay) with lr = 1e-5. Train the model for 10 epochs or so.
   3. Choose an appropriate checkpoint as the starting point(best ckpt for example). Unfreeze all the parameters.  Choose a proper scheduler and optimizer (AdamW with weight decay) with lr = 1e-5. Train the model for 10 epochs or so.
